### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by mancuoj**

### CODE REVIEW

**Comment by mancuoj** [comment]
By the way, after completing the tutorial, I have a question: why after signing up, I sign in and then check the profile, an error appears, saying there is no token? I used the final code in tutorial.

File: docs/tutorial.md [hunk 1]
```diff
@@ -1821,11 +1821,6 @@ import { user } from './user' // [!code ++]
 
 const app = new Elysia()
     .use(swagger())
-    .onError(({ error, code }) => {
```
Comment (bogeychan) [inline docs/tutorial.md:1824]: why did you remove this?

Comment (mancuoj) [reply docs/tutorial.md:1824]: Because this new code is mentioned in the following article

Comment (mancuoj) [reply docs/tutorial.md:1824]: There is no explanation for this part. The following is an explanation of this addition.

**Comment by bogeychan** [comment]
> By the way, after completing the tutorial, I have a question: why after signing up, I sign in and then check the profile, an error appears, saying there is no token? I used the final code in tutorial.

You have to:

* Create a user: `PUT http://localhost:3000/user/sign-up`, body: `{ "username": "42", "password": "123456789" }`
* Login with user: `POST http://localhost:3000/user/sign-in`, body: `{ "username": "42", "password": "123456789" }`
* And use the cookies from user login to get note: `GET http://localhost:3000/note`
